zuul.d/nodeset.yaml

```diff
@@ -44,31 +44,12 @@
         label: centos-9-stream-crc-2-30-0-xxl
 
 - nodeset:
-    name: centos-9-rhel-9-2-crc-extracted-2.30-3xl
+    name: centos-9-multinode-rhel-9-2-crc-extracted-2-36-0-3xl-novacells
```

Comment:
nit but this is a 2.36 nodeset in the 2.30 section, there is a chance that when the 2.30 nodesets are removed, the whole section is deleted, without realising that it's deleting a 2.36 nodeset in the process

Reply:
ack ok let me move that sec